src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/VedtaksperioderMedBegrunnelser/FritekstVedtakbegrunnelser.tsx

```diff
@@ -44,7 +44,7 @@ const StyledLabel = styled(Label)`
 `;
 
 const InfoBoks = styled.div`
-    margin-right: 5.6875rem;
+    margin-right: 5.85rem;
```

Comment:
For å få tagens høyrekant til å ligge på linje med tekstfeltet
<img width="256" alt="Screenshot 2023-03-20 at 12 09 02" src="https://user-images.githubusercontent.com/2379098/226558113-3625b34d-9ab8-4c27-9785-490011c95b78.png">

Reply:
🙌